dmutils/content_loader.py

```diff
@@ -432,6 +458,20 @@ def _populate_section(self, framework_slug, question_set, section):
         ]
         return section
 
+    def _message_key(
+        self, block, framework_status, supplier_status
+    ):
+        return '{}-{}{}'.format(
+            block,
+            framework_status,
+            '-{}'.format(supplier_status) if supplier_status else ''
+        )
+
+    def _message_error(self, framework_slug, message):
+        self._content[framework_slug]['messages'][message] = None
```

Comment:
Intention is that if there isn't a file for a specific block and status then the front end won't render anything. So the app needs to be able to ask for a file which doesn't exist without throwing an exception. And we should cache this as `None` so that it doesn't make a disk operation every time it does this.